src/xc/xc_evaluate.jl

```diff
@@ -7,28 +7,54 @@ end
 
 include("lda_x.jl")
 include("lda_c_vwn.jl")
+include("lda_c_pw.jl")
+include("gga_x_pbe.jl")
+include("gga_c_pbe.jl")
 
 # Function that always dispatches to the DFTK fallback implementations
 function xc_fallback!(func::Functional, ::Val{:lda}, ρ::AbstractArray;
                       zk=nothing, vrho=nothing, v2rho2=nothing)
     func.n_spin == 1  || error("Fallback functionals only for $(func.n_spin) == 1")
     zk = reshape(zk, size(ρ))
 
-    fE = ρ -> energy_per_particle(Val(func.identifier), ρ)
-    if zk !== nothing
+    fE(ρ) = energy_per_particle(Val(func.identifier), ρ)
```

Comment:
I was wary of doing this because of the super nasty https://github.com/JuliaLang/julia/issues/15602, but OK it's not an issue here if we don't redefine it.